halo2_gadgets/src/utilities/lookup_range_check.rs

```diff
@@ -77,107 +85,35 @@ impl<F: PrimeFieldBits, const K: usize> LookupRangeCheckConfig<F, K> {
     /// # Side-effects
     ///
     /// Both the `running_sum` and `constants` columns will be equality-enabled.
-    pub fn configure(
+    fn configure(
```

Comment:
the documentation should be part of the trait or the implementation? 
I think implementation.